Content.Shared/Item/ItemToggle/Components/ComponentTogglerComponent.cs

```diff
@@ -29,4 +29,7 @@ public sealed partial class ComponentTogglerComponent : Component
     /// </summary>
     [DataField]
     public bool Parent;
+
+    [DataField]
+    public EntityUid? Target = null;
```

Comment:
```suggestion
    public EntityUid? Target;
```
nullable types default to null, so no need to write it out
also add documentation

Reply:
fixed